CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java

```diff
@@ -41,31 +46,56 @@
 @NbBundle.Messages("SaveGraphPlugin=Save Graph")
 public class SaveGraphPlugin extends SimplePlugin {
 
+    private static final Logger LOGGER = Logger.getLogger(SaveGraphPlugin.class.getName());
+
     public static final String GRAPH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, "graphId");
+    public static final String FILE_PATH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, "saveFilePathId");
 
     @Override
     public PluginParameters createParameters() {
         final PluginParameters parameters = new PluginParameters();
 
         final PluginParameter<StringParameterValue> graphIdParameter = StringParameterType.build(GRAPH_PARAMETER);
+        final PluginParameter<StringParameterValue> filePathParameter = StringParameterType.build(FILE_PATH_PARAMETER);
+
         graphIdParameter.setName("graphId");
         graphIdParameter.setDescription("The Id of the graph");
         parameters.addParameter(graphIdParameter);
 
+        filePathParameter.setName("saveFilePath");
```

Comment:
It doesn't matter too much in this case - but generally you want parameter names to be structured as human readable text, ie. "Save File Path".